application/src/test/java/org/opentripplanner/updater/trip/TripUpdateBuilder.java

```diff
@@ -37,32 +38,138 @@ public TripUpdateBuilder(
     this.midnight = ServiceDateUtils.asStartOfService(serviceDate, zoneId);
   }
 
-  public TripUpdateBuilder addStopTime(String stopId, int minutes) {
+  public TripUpdateBuilder(
+    String tripId,
+    LocalDate serviceDate,
+    GtfsRealtime.TripDescriptor.ScheduleRelationship scheduleRelationship,
+    ZoneId zoneId,
+    String tripHeadsign,
+    String tripShortName
+  ) {
+    this(tripId, serviceDate, scheduleRelationship, zoneId);
+    tripUpdateBuilder.setTripProperties(
+      GtfsRealtime.TripUpdate.TripProperties.newBuilder()
+        .setTripHeadsign(tripHeadsign)
+        .setTripShortName(tripShortName)
+        .build()
+    );
+  }
+
+  public TripUpdateBuilder addStopTime(String stopId, int secondsFromMidnight) {
```

Comment:
```suggestion
  public TripUpdateBuilder addStopTime(String stopId, String time) {
```

It's my mistake for not introducing this earlier but you can use time strings like "10:00" and convert to seconds from midnight with `TimeUtils.time`.